CONTRIBUTING.md

```diff
@@ -63,21 +63,47 @@ When reporting an issue, details are key. Include the following:
+* **reviewed/wont-fix** - the universal sign that we won't fix this issue. This tag is important to use as we separate out the nice-to-have features from the maintainer's agreed upon strategic direction
+* **wip-do-not-merge** - specifically for a PR to indicate it is not ready for review just yet
+
+The difference between bugs, features and enhancements can be confusing. To be extra clear, we reduced it down to two options. Here are their definitions:
+* **type/bug** - the classic definition of missing or misbehaving code from existing functionality (this includes malfunctioning tests)
+* **type/feature-or-enhancement** - any new functionality or improvements/enhancements to existing functionality. We use one label because it's prioritized in identical ways during sprint planning
+
+For the sake of clarity, here are a few scenarios you might see play out.
+
+As a maintainer:
+* An issue is opened stating that Snap is not working. Upon review, the maintainer finds it is an issue with a plugin. She will label the issue with **reviewed/wrong-repo** and open a new issue under the plugin where she tags the original issue reporter, links the original issue and labels it **bug** (which is available in plugins repositories).
+* An issue is opened stating that Snap is not working. It turns out to be related to a Snap's functionality. The maintainer will label it **type/bug**. She takes on the bug, labels in **reviewed/in-progress**. She finds it maps to the Scheduler package and adds additional context with **component/scheduler**. She opens a PR while she fixes this issue. In the PR body, it says "Fixes #" for the previous issue and labels it **wip-do-not-merge**.
+* As PR is opened that will change Snap functionality (examples at [#977](https://github.com/intelsdi-x/snap/issues/977) & [#803](https://github.com/intelsdi-x/snap/issues/803)). A maintainer labels it **reviewed/needs-2nd-review** and proceeds with the normal code review. If the initial maintainer labels LGTM, another maintainer must review it again. A discussion must take place with a technical lead before merging.
+* A PR is opened which changes the metadata structure for a plugin. A maintainer labels it **reviewed/needs-2nd-review** and adds whatever **breaking-change/** labels are appropriate. If the initial maintainer labels LGTM, another maintainer must review it again. A discussion must take place with a technical lead before merging. This corresponding issue is added to a milestone that corresponds with its targeted release (real example at [#871](https://github.com/intelsdi-x/snap/issues/871)).
+* A PR is opened that edits a small amount of markdown or string output text. A maintainer labels it **effort/small**, gives it a quick review to ensure it renders, writes LGTM and merges it themselves (example: [#1139](https://github.com/intelsdi-x/snap/issues/1136)).
```

Comment:
To be consistent, an example should point to a pull request - consider using some of these [PRs effor/small]( https://github.com/intelsdi-x/snap/pulls?q=is%3Apr+is%3Aclosed+label%3Aeffort%2Fsmall)